docs/02.API-REFERENCE.md

```diff
@@ -523,6 +546,36 @@ main (void)
 - [jerry_cleanup](#jerry_cleanup)
 
 
+## jerry_get_memory_stats
+
+**Summary**
+
+Get heap memory stats.
+
+**Prototype**
+
+```c
+bool
+jerry_get_memory_stats (jerry_heap_stats_t *out_stats_p);
+```
+
+- `out_stats_p` - out parameter, that gives the stats of the heap.
+- return value
+  - true, if run was successful
+  - false, otherwise. Usually it is because the MEM_STATS feature is not enabled.
+
+**Example**
+
+```c
+jerry_heap_stats_t stats = {0};
```

Comment:
Sorry, a little confused. Do you mean the `version` is a input param and  `jerry_get_mem_stats` should check the `version`? But in this way, user must know the version clearly and explicitly. I doubt if it is a  good way.